### DISCUSSION TITLE
How to get my change into current version

### DISCUSSION THREAD

**Discussion by Wallabeng** [vmware/clarity#5155] [Q&A] [answered]
First of all sorry, that I submit an issue for that, but don't know where to ask that elsewhere:
My PR #5070 was merged some time ago and I expected it to be in the next release.
Now I saw that it's not in latest 4.0.4, so I ask you how I get this change into a release?
And what's the point of having a separate branch for the latest version (out of curiosity)?

**Comment by bbogdanov** [comment]
Because of our workflow after the change goes into `next`, it needs to be backported to the `v4` branch in order to be released in the incoming release.

**Comment by mathisscott** [comment, marked as answer]
Hi @Wallabeng 
This is my bad. I was the release engineer that month. I merged your PR but did not backport it to v4. 
@hippee-lee  , could you backport this PR to v4? I can do it if you are short on time.

**Reply by hippee-lee** [reply]
No problem. I can back port it. It will be slated to go out in the 4.0.5 release during the next cycle.

**Comment by Wallabeng** [comment]
Alright, thanks for clarification and backporting!
